scripts/hostcfgd

```diff
@@ -1553,18 +1546,13 @@ class MgmtIfaceCfg(object):
                 cmd0 = ['cat', '/proc/net/route']
                 cmd1 = ['grep', '-E', r"eth0\s+00000000\s+[0-9A-Z]+\s+[0-9]+\s+[0-9]+\s+[0-9]+\s+202"]
                 cmd2 = ['wc', '-l']
-                run_cmd_pipe(cmd0, cmd1, cmd2, True, True)
-            except subprocess.CalledProcessError:
-                syslog.syslog(syslog.LOG_ERR, 'MgmtIfaceCfg: Could not delete '
-                                              'eth0 route')
+                check_output_pipe(cmd0, cmd1, cmd2)
+            except Exception as err:
```

Comment:
Should we handle other exception scenarios as well? I believe this may not be the only exception

Reply:
Updated